docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -108,13 +108,41 @@ The Appendix for the GBD 2017 Cause of Death manuscript is also available and Op
 Modeling non-fatal outcomes (Outline)
 ---------------------------
 
-- Figure of process
 .. image:: gbd_nonfatal_flowchart_sim_team.jpg
    :width: 600
 
-**Modeling approach** 
-
-- Modeling tools like DisMod, ST-GPR
+**DisMod** 
+
+Disease Model Meta Regression 2.1 (DisMod, sometimes DisMod MR) is a statistical modeling tool developed for the Global
+Burden of Disease study to estimate non-fatal disease burden. It is the most frequently used tool for non-fatal modeling
+in the GBD and models that are run outside of it are frequently called *custom models*. 
+
+DisMod has a user interface. The 
+`External Version
+<https://vizhub.healthdata.org/epi/>`_
+shows published and final models for GBD 2017.
+The 
+`Internal Version
+<https://internal.ihme.washington.edu/epi/>`_
+is where new and ongoing modeling occurs and is for GBD 2019 (but shows GBD 2017 best models). 
+
+There are three main components to DisMod. The first is that it
+is a meta-regression statistical model. This means that it uses point estimates with uncertainty around those estimates and covariates
+to predict disease burden for every location/year/age/sex. The second is that it is a compartmental model of disease. This
+means that DisMod is solving differential equations while fitting meta-regression estimates to enforce consistency between
+disparate measures of disease like incidence, prevalence, remission, and excess mortality. The third is that DisMod is age-integrating.
+Input data in DisMod must have an age or age range associated with those data. DisMod can account for ranges in age in the input
+data to help it produce continuous estimates of disease burden across all ages.
+
+Input data for DisMod can be any of the measures of disease that are estimated within it. These include prevalence, incidence,
+remission, excess mortality, and cause-specific mortality. Input data will be described in more detail in the following section.
+
+Results from DisMod are internally consistent *within* that model. Incidence, prevalence, remission, and excess mortality are
+linked in the DisMod estimation process and so the results from a model will include all these measures of disease. However, results
+from DisMod will *not* be consistent with final GBD estimates because of processes like COMO that rescale prevalence and sometimes
+incidence of non-fatal disease models. 
```

Comment:
maybe add something like, "because of post-processes like COMO that rescale prevalence and sometimes incidence of different non-fatal models in order to achieve consistency across disease models."?